brainbox/examples/docs_wheel_moves.ipynb

```diff
@@ -76,7 +76,7 @@
     "\n",
     "For more information on the rotary encoder see these links:  \n",
     "[National Instruments guide to rotary encoders](http://www.ni.com/tutorial/7109/en/)  \n",
-    "[Datasheet for the KÃƒÂ¼bler](https://www.kuebler.com/pdf?2400-2420_en.pdf)"
+    "[Datasheet for the KÃƒÆ’Ã†â€™Ãƒâ€ Ã¢â‚¬â„¢ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¼bler](https://www.kuebler.com/pdf?2400-2420_en.pdf)"

```

Comment:
This looks like a character encoding error that's progressively getting worse.  It was originally a 'u' with an umlaut. Maybe change it to a u or change the encoding of the notebook?  Option 1 is probably easier.  Also I noticed that the notebook it no longer run before saving, meaning when a user visits the notebook page on GitHub they don't see any of the plots, etc.  Is this intended?

Reply:
I have corrected these, encoding errors. Thanks for spotting. Regarding notebook being run, I have made the code to unexecute all notebooks so we don't keep run notebooks in the repo. The run example of the document is on the docs website https://int-brain-lab.github.io/iblenv/notebooks_external/docs_wheel_moves.html could you redirect people here.